drivers/gles3/shader_gles3.cpp

```diff
@@ -268,10 +269,17 @@ void ShaderGLES3::_get_uniform_locations(Version::Specialization &spec, Version
 		}
 	}
 	// textures
-	for (int i = 0; i < p_version->texture_uniforms.size(); i++) {
-		String native_uniform_name = _mkid(p_version->texture_uniforms[i]);
+	int texture_index = 0;
+	for (uint32_t i = 0; i < p_version->texture_uniforms.size(); i++) {
+		String native_uniform_name = _mkid(p_version->texture_uniforms[i].name);
 		GLint location = glGetUniformLocation(spec.id, (native_uniform_name).ascii().get_data());
-		glUniform1i(location, i + base_texture_index);
+		PackedInt32Array texture_uniform_bindings;
```

Comment:
This can use a ``Vector<int32_t>`` instead, then you don't need to include variant.h. ``PackedInt32Array`` is just a typedef ``Vector<int32_t>`` anyway

Reply:
Thanks, updated the code according to your suggestions.